### PR TITLE
CLDR-15003 Keyboards: Add a note to TR35§7

### PR DESCRIPTION
**Pull request by srl295**
- Inform people that the Keyboard SC is at work
- Mention CLDR-15034

CLDR-15003

- [ ] This PR completes the ticket.

### CODE REVIEW

**Comment by srl295** [comment]
Review Note: please ignore whitespace

**Comment by miloush** [comment]
I think "major changes" instead of "many changes" might reflect better what to anticipate, but otherwise LGTM.

**Comment by srl295** [comment]
@miloush good point, changed 'many' to 'major'

File: docs/ldml/tr35-keyboards.md [hunk 1]
```diff
@@ -10,6 +10,12 @@
 
 For the full header, summary, and status, see [Part 1: Core](tr35.md).
 
+#### _Important Note_
+
+> The CLDR [Keyboard Workgroup](https://cldr.unicode.org/index/keyboard-workgroup) is currently
+> developing major changes to the CLDR keyboard specification. These changes are targetted for
```
Comment (mcdurdin) [inline docs/ldml/tr35-keyboards.md:16]:
```suggestion
> developing major changes to the CLDR keyboard specification. These changes are targeted for
```

"targetted" is not technically illegal but it is certainly unusual...

"Major" or "significant"?  But we can nit pick forever.

Looks Good To Me.

Comment (srl295) [reply docs/ldml/tr35-keyboards.md]: thanks. @mcdurdin changed it to targeted

**Review by mcdurdin — APPROVED** [review]

**Review by macchiati — APPROVED** [review]

**Pull request merged**